pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1336,6 +1337,13 @@ public void onUpdate(String path, Map<String, String> data, Stat stat) {
         return dynamicConfigurationMap.keys();
     }
 
+    public List<String> getRuntimeConfiguration() {
```

Comment:
Should this return a map instead of a list? Or maybe should the values be sorted so it's easier to read with printing from the cli?

Reply:
Just sort the list result, thanks!